test/hotspot/jtreg/runtime/ClassFile/ClassFileVersionTest.java

```diff
@@ -28,23 +28,34 @@
  * @summary Test getting the class file version for java.lang.Class API
  * @modules java.base/java.lang:open
  * @compile classFileVersions.jcod
+ * @compile --enable-preview -source ${jdk.version} ClassFileVersionTest.java
  * @run main/othervm --enable-preview ClassFileVersionTest
  */
 
 import java.lang.reflect.*;
 
 public class ClassFileVersionTest {
-
+    public static final int LOWER_16 = 0x0000_FFFF;
+    /*
+     * Include a use of a preview API so that the minor class file
+     * version of the class file for this class gets set during
+     * compilation. If a particular class becomes non-preview, any
+     * currently preview class can be substituted in.
```

Comment:
This potentially will need changing on each release so don't really make the test more stable.